live-examples/js-examples/intl/intl-locale.html

```diff
@@ -0,0 +1,12 @@
+<pre>
+<code id="static-js">const korean = new Intl.Locale("ko", {script: "Kore", region: "KR", hourCycle: "h24", calendar: "gregory"});
+const japan = new Intl.Locale("ja-Jpan-JP-u-ca-japanese-hc-h12")
```

Comment:
Also a blank space before this line (to separate the two declarations) would help readability I think.